budgettool/transaction.py

```diff
@@ -35,6 +35,7 @@ def __iter__(self):
             ('name',     self.transaction.name),
             ('category', self.transaction.category),
             ('amount',   self.transaction.amount),
+            ('schedule', { 'type': self.schedule.schedule_type, 'data': dict(self.schedule) } ),
```

Comment:
Should construct full schedule dictionary inside schedule classes.

transactions shouldn't know the details of a schedule.